src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int NUMBER_OF_FIGURE = 6;
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[NUMBER_OF_FIGURE];
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (int i = 0; i < NUMBER_OF_FIGURE / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+        for (int i = NUMBER_OF_FIGURE / 2; i < NUMBER_OF_FIGURE; i++) {
```

Comment:
Creating a new figure in each iteration of the loop is fine, but ensure that the 'FigureSupplier' instance is not being created inside the loop (which is not the case here, so it's good).